tsconfig.json

```diff
@@ -3,5 +3,8 @@
   "extends": "./.nuxt/tsconfig.json",
   "compilerOptions": {
     "types": ["vitest/globals"]
-  }
+  },
+  // Need this to not have conflicts between `expect` and `assert` from Cypress and Vitest
+  // https://github.com/cypress-io/cypress/issues/22059
+  "exclude": ["cypress.config.js"]
```

Comment:
Would not relying on global functions solve this?
I would prefer importing the vitest functions, it's more explicit. So in the test files we'd put `import { test, etc } from "vitest"`.

Reply:
I personally quite like globals/auto-imports for tests to not make it annoying (especially in a Nuxt project which auto-imports so many other things) + it meant less code changes when I migrated to Vitest from Jest. But if there's a common way to work with that in De Voorhoede, I'd gladly comply.

Reply:
Handling it in another PR